src/IO/HDF5/ParallelHDF5IOHandler.cpp

```diff
@@ -73,16 +73,65 @@ ParallelHDF5IOHandlerImpl::ParallelHDF5IOHandlerImpl(AbstractIOHandler* handler,
 
     herr_t status;
     status = H5Pset_dxpl_mpio(m_datasetTransferProperty, xfer_mode);
+    VERIFY(status >= 0, "[HDF5] Internal error: Failed to set H5Pset_dxpl_mpio");
 
-    auto const strByte = auxiliary::getEnvString( "OPENPMD_HDF5_ALIGNMENT", "1" );
+    auto const strByte = auxiliary::getEnvString( "OPENPMD_HDF5_ALIGNMENT", "4194304" );
     std::stringstream sstream(strByte);
     hsize_t bytes;
     sstream >> bytes;
 
+    /*
+     * Note from https://support.hdfgroup.org/HDF5/doc/RM/RM_H5P.html#Property-SetCache:
+     * "Raw dataset chunk caching is not currently supported when using the MPI I/O
+     * and MPI POSIX file drivers in read/write mode [...]. When using one of these
+     * file drivers, all calls to H5Dread and H5Dwrite will access the disk directly,
+     * and H5Pset_cache will have no effect on performance."
+     */
+    int metaCacheElements = 0;
+    size_t rawCacheElements = 0;
+    size_t rawCacheSize = 0;
+    double policy = 0.0;
+    status = H5Pget_cache(m_fileAccessProperty, &metaCacheElements, &rawCacheElements, &rawCacheSize, &policy);
+    VERIFY(status >= 0, "[HDF5] Internal error: Failed to set H5Pget_cache");
+    rawCacheSize = bytes * 4; // default: 1 MiB per dataset
```

Comment:
should be guarded so we don't accidentally set 1Byte cache if a user provides `1`